cuizinart/frontend/src/components/map.jsx

```diff
@@ -63,6 +63,17 @@ export default class MapComp extends Component {
         this.setState({ currentZoomLevel: newZoomLevel });
     }
 
+    callbackUpdate() {
```

Comment:
Will the filtering also be fired when editing a shape? I see in `_onEdited` you call `_onChange`, but does that update the filter?

Reply:
Yes, they call the same function which contains the filter.